src/mailing/sender.js

```diff
@@ -1,7 +1,7 @@
 const nodemailer = require('nodemailer');
 const Email = require('email-templates');
 const path = require('path');
-
+const ejs = require('ejs');
```

Comment:
Is this import needed?

Reply:
@akosasante no, deleted it, thank you!!!